### PR TITLE
Added launch alarm for PR #4976

### PR DESCRIPTION
**Pull request by fordmichigan**
# About the pull request

Adds a looping alarm sound when the lifeboats are launched. This also uses the function described in PR #4976 to trigger the sound. This is NOT for adding those features, so wait until that is merged to merge this PR.

# Explain why it's good for the game

Denies the need to make an announcement warning marines that the lifeboats are leaving. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

https://youtu.be/L8ia2OKhUUA

</details>

# Changelog

:cl:alexguinea atticus-rezzer
add: Adds alarm sound for lifeboats when launching
/:cl:

### CODE REVIEW

**Comment by zzzmike** [comment]
Hi, 4976 author here. I think the ship announcement is good _in addition_ so that you'll know whether the port or starboard boat is the one launching - I assume the sound will go far enough that it isn't obvious which boat it's coming from at first. Also, thank you!! Ever since the dropship alarm got merged, I was thinking of stealing it for my PR and never got around to it, heh. ❤️

File: code/modules/shuttle/computer.dm [hunk 1]
```diff
@@ -293,23 +295,42 @@
...
+					to_chat(user, SPAN_NOTICE("[src]'s screen blinks and says \"Launch sequence already initiated\"."))
+					return
+				switch(response)
+					if ("Yes")
+						launch_initiated = TRUE
+						to_chat(user, "[src]'s screen blinks and says \"Launch command accepted\".")
+						shipwide_ai_announcement("Launch command received. [lifeboat.id == MOBILE_SHUTTLE_LIFEBOAT_PORT ? "Port" : "Starboard"] Lifeboat doors will close in 10 seconds.")
+						addtimer(CALLBACK(lifeboat, TYPE_PROC_REF(/obj/docking_port/mobile/crashable/lifeboat, evac_launch)), 10 SECONDS)
+						lifeboat.alarm_sound_loop.start()
+						lifeboat.playing_launch_announcement_alarm=TRUE
```
Comment (zzzmike) [inline code/modules/shuttle/computer.dm:325]: I would go lifeboat.playing_launch_announcement_alarm = TRUE here for spacing and for 🤓 - other than that looks great, thanks again.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by fordmichigan** [comment]
How do I fix the conflicts?

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Drulikar — APPROVED** [review]

**Pull request merged**